api/functions/queries.js

```diff
@@ -289,6 +289,17 @@ exports.getUnscannableLinks = () =>
 
 exports.compareScans = (api, url) =>
 	new Promise(async (resolve) => {
+		// Standardize url string
+		if (!url.startsWith("https://")) {
```

Comment:
Do we need to worry about non-https (http://) links here?

Reply:
Yes we do - I have added extra condition to check for non-https
Thanks for the feedback!